### ISSUE TITLE
Tons of wrong size items

### ISSUE THREAD

**Issue by TealcOneill** [CleverRaven/Cataclysm-DDA#50402]
### Describe the bug

All arrows are 1 inch long.
Almost all ammunition has a volume of 0.01.
Almost all ammunition shows no length or a length of 0.

### Steps To Reproduce

I'm unsure how this happened. I simply loaded up the game the other day and noticed arrows were 1 inch and then saw almost everything in the Ammo tab of the crafting menu were weirdly sized.

### Expected behavior

I used https://nornagon.github.io/cdda-guide/#/item, to search the proper sizes and none of them match.
Arrows are supposed to be close 20 inches.
Ammo usually has a higher volume and some length.

### Screenshots

![ammo](https://user-images.githubusercontent.com/45678006/127786438-79bdae6f-1c38-43bc-b5b3-3e6b5d5de07a.png)
![ammo2](https://user-images.githubusercontent.com/45678006/127786441-c4502dd7-907a-420e-93fd-147549fcbb89.png)
![ammo3](https://user-images.githubusercontent.com/45678006/127786443-f81b9074-9964-46d5-a5fa-569e146ede75.png)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0 2009
- Game Version: 0.E-5059-g10b2315 [64-bit]
- Graphics Version: Tiles
- Game Language: System language [English]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food]
]

### Additional context

[save.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/6913510/save.zip)

**Comment by Saicchi** [comment]
You're playing on 0.E ( 0.E experimental? ) which is very old and seems to be before #43424 was merged. Can you reproduce this issue with a newer version? Try 0.F-stable or the experimental releases.

**Comment by TealcOneill** [comment]
Well, I've confused now. I should be running the latest stable version 0.F. I didn't pay attention to that when I copied it since I assumed it would be 0.F. However I just updated the launcher and it's now saying it's this. 
- OS: Windows
    - OS Version: 10.0 2009
- Game Version: bc28921 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food]
]
It might be an issue with the fact that I have the old experimental I used to play on and the Launcher in the same file. Checking now and most of the sizes seem to be fixed. I think I'm going to delete the whole lot and reinstall the game from scratch because this might be causing issues.
